README.md

```diff
@@ -107,6 +106,153 @@ print(myShoppingList)
 // prints "["Almond Butter", "Jelly", "Bread", "Strawberries", "Bananas"]"
 ```
 
-* Challenge the student to write a function that takes in an `Array` as an argument, the `Array`'s type is `[String]`, where they are asked to grab the first element in the array (at index 0) and see if it's value is equal to "Bread". If so, return `true`.. if not, return `false`.
+- Challenge the student to write a function that takes in an `Array` as an argument, the `Array`'s type is `[String]`, where they are asked to grab the first element in the array (at index 0) and see if it's value is equal to "Bread". If so, return `true`.. if not, return `false`.
+
+--------------------------------------------------------------------------------
+
+## Arrays
+
+So far you've used variables to store `String` and `Int` values. For example, you could create a variable to represent an ingredient:
+
+```swift
+let ingredient = "Jalapeno"
+```
+
+Variables only allow you to store one item of information at a time. What if we were making a shopping list and needed all the ingredients wanted to make a sandwich?
+
+You could create multiple variables to represent the ingredients:
+
+```swift
+let ingredient1 =  "Bread"
+let ingredient2 = "Butter"
+let ingredient3 = "Cheese"
+let ingredient4 = "Lettuce"
+let ingredient5 = "Tomatoes"
+```
+
+And then write code that passes these variables one at a time. This will get messy quickly. What if you forget an ingredient or need to add a new one? In the real world, you would write down all your ingredients on one shopping list. With Swift, like most programming languages, you achieve the same using an _array_. An array allows a variable to contain a _collection_ of values of the same type.
+
+Arrays share similar concepts to other collection types available in Swift and you will cover these later in the course. For now it's important to know that an array is an _ordered_ collection of values, i.e. that every value has a particular place (or _index_) in the collection.
+
+## Creating an Array
+
+Here's how to create an array of values in Swift which represents the ingredients of a sandwich.
+
+```swift
+var list = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+In this case, you are creating a variable with the `var` keyword called 'list'. Swift can infer that `[String]` is the correct type for the `list` variable, based on it's content.
+
+If you option click the `list` variable you can check Swift's guess.
+
+![Inferred Type](inferred_type.png)
+
+To be more precise, you can declare the type of values allowed in the array explicitly:
+
```

Comment:
Now that we feel like the story is great, we're just being a little more nit-picky with specific words.

I think we can add onto this a little and state something like:

If you wanted to be more precise when creating an array, you can declare the type of values allowed in the array explicitly: